api/src/main/java/com/velocitypowered/api/network/ProtocolVersion.java

```diff
@@ -6,11 +6,12 @@
 import java.util.HashMap;
 import java.util.Map;
 import java.util.Set;
+import net.kyori.coffee.Ordered;
 
 /**
  * Represents each Minecraft protocol version.
  */
-public enum ProtocolVersion {
+public enum ProtocolVersion implements Ordered<ProtocolVersion> {
```

Comment:
I think we'll probably convert `ProtocolVersion` into an interface. Implementing `Comparable<ProtocolVersion>` (okay, a subinterface) is a good move.